apps/sms/style/sms.css

```diff
@@ -443,7 +443,7 @@ section[role="region"].new > header:first-child form {
 */
 
 .carrier-wrapper {
-  display: none;
+  display: block;
```

Comment:
cases where it should be hidden: group MMS, new message panel, select mode (maybe I forget some)